projects/kit/components/combo-box/combo-box.directive.ts

```diff
@@ -122,6 +122,15 @@ export class TuiComboBox<T>
         }
     }, TUI_ALLOW_SIGNAL_WRITES);
 
+    protected readonly stringifyEffect = effect(() => {
+        this.itemsHandlers.stringify();
```

Comment:
This line is unnecessary. And let's rename itemsHandlers to handlers for brevity.